node/malus/Cargo.toml

```diff
@@ -37,6 +37,7 @@ rand = "0.8.5"
 
 [features]
 default = []
+fast-runtime = ["polkadot-cli/fast-runtime"]
```

Comment:
Is this needed?

Reply:
I'm not 100% sure anymore, but it doesn't hurt either :rofl: